docs/graphcache/custom-updates.md

```diff
@@ -5,8 +5,10 @@ order: 3
 
 # Custom Updates
 
-_Graphcache_ will attempt to automatically react to your mutations' and subscriptions' results
-but sometimes this isn't possible. While it will update all normalized entities that it finds in
+Every time a `subscription` triggeres or a `mutation` receives a response GraphCache will look at the response and see
+if it can get a key from the response. This key is by default derived from the `__typename` and `id`/`_id`, unless
+customized in your `keys` configuration, this entity will then be added to the cache this means that
+an existing entity with the found key will be updated.. While it will update all normalized entities that it finds in
```

Comment:
```suggestion
This means that existing entities with a found key will be updated and new ones will be written. While it will update all normalized entities that it finds in
```